Source/SwiftyDropbox/Shared/Handwritten/OAuth/OAuth.swift

```diff
@@ -31,6 +33,7 @@ open class DropboxOAuthManager {
     /// Session data for OAuth2 code flow with PKCE.
     /// nil if we are in the legacy token flow.
     var authSession: AuthSession?
+    weak var sharedApplication: SharedApplication?
```

Comment:
why is this weak?

Reply:
OAuthManager doesn't have to own SharedApplication, so I made it weak.